SECURITY.md

```diff
@@ -0,0 +1,6 @@
+
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report security vulnerabilities, please go to https://pivotal.io/security.
```

Comment:
What if it's a security vulnerability in a Google product that the code integrates with?

Reply:
Then why would they be reporting the issue to us and not Google?

Reply:
Because this document is telling them? :smile: 
Maybe we should be more clear here to say that this is about reporting security vulnerabilities in the Spring Cloud? I don't know. Maybe I'm just overthinking this.